tests/version.py

```diff
@@ -287,5 +298,31 @@ def write_ks_cfg(buf):
         self.assertRaises(KickstartVersionError, versionFromFile, ks_cfg)
         os.unlink(ks_cfg)
 
+class FailedImpImport_TestCase(CommandTest):
+    def runTest(self):
+        try:
+            import sys
+
+            # will force another import
+            del sys.modules['pykickstart.version']
+            del sys.modules['imp']
+
+            # will cause ImportError
+            sys.modules['imputil'] = None
+
+            import pykickstart.version as ver
+            from pykickstart.handlers.f23 import F23Handler
+
+            cls = ver.returnClassForVersion(ver.F23)
+
+            # assert the names; b/c of how the imp.load_module() works
+            # asserting both classes being equal fails
+            self.assertEqual(cls.__name__, F23Handler.__name__)
+        finally:
+            # force import to reload these modules
+            del sys.modules['pykickstart.version']
+            del sys.modules['imp']
+            del sys.modules['imputil']
```

Comment:
I think this needs to import the modules back because tests that run after this one may need it (and if they are in the same module, they wouldn't do the import on their own, I think).